wolfcrypt/src/rabbit.c

```diff
@@ -149,12 +149,15 @@ static WC_INLINE int DoKey(Rabbit* ctx, const byte* key, const byte* iv)
 {
     /* Temporary variables */
     word32 k0, k1, k2, k3, i;
+    word32 keyc[4];
+
+    XMEMCPY(keyc, key, sizeof(keyc));
```

Comment:
`memcpy` wrap in `XSTREAM_ALIGN`?

Reply:
Alignment was fixed in wc_RabbitSetKey().